R/build_lm.R

```diff
@@ -1,3 +1,27 @@
+calc_permutation_importance_logistic <- function(fit, target, vars, data) {
+  var_list <- as.list(vars)
+  importances <- purrr::map(var_list, function(var) {
+    mmpf::permutationImportance(data, var, target, fit, nperm = 1, # By default, it creates 100 permuted data sets. We do just 1 for performance.
+                                predict.fun = function(object,newdata){predict(object,type = "response",newdata=newdata)},
+                                # Use minus log likelyhood as loss function, since it is what logistic regression tried to optimise. 
+                                loss.fun = function(x,y){-sum(log(1- abs(x - y)),na.rm = TRUE)})
+  })
+  importances <- purrr::flatten_dbl(importances)
+  importances_df <- tibble(term=vars, importance=importances)
+  importances_df
+}
+
+calc_permutation_importance_linear <- function(fit, target, vars, data) {
+  var_list <- as.list(vars)
+  importances <- purrr::map(var_list, function(var) {
+    mmpf::permutationImportance(data, var, target, fit, nperm = 1, # By default, it creates 100 permuted data sets. We do just 1 for performance.
```

Comment:
Add a description why `predict.fun` is missing here.

Reply:
Done.